backend/src/main/scala/cromwell/backend/io/DirectoryFunctions.scala

```diff
@@ -29,19 +28,21 @@ trait DirectoryFunctions extends IoFunctionSet with PathFactory {
 
   override def isDirectory(path: String) = Future.fromTry(Try(buildPath(path).isDirectory))
 
-  override def listDirectory(path: String): Future[Iterator[String]] = Future.fromTry(Try(buildPath(path.ensureSlashed).list.map(_.pathAsString)))
+  override def listDirectory(path: String)(visited: Vector[String] = Vector.empty): Future[Iterator[IoElement]] = {
+    Future.fromTry(Try {
+      val visitedPaths = visited.map(buildPath)
+      val cromwellPath = buildPath(path.ensureSlashed)
 
-  override def listAllFilesUnderDirectory(dirPath: String): Future[Seq[String]] = {
-    temporaryImplListPaths(dirPath)
-  }
```

Comment:
actually I'm using this in my 87 branch... 🤔

Reply:
Oh, I'll put it back then